GETTING_STARTED.md

```diff
@@ -0,0 +1,602 @@
+```
+
+And add this content:
+
+main.tf
+```
+provider "aws" {
+  access_key = "${var.access_key}"
+  secret_key = "${var.secret_key}"
+  region = "${var.region}"
+}
+
+resource "aws_instance" "example" {
+  ami           = "${var.ami}"
+  instance_type = "${var.instance_type}"
+  key_name      = "${var.key_name}"
+}
+```
+
+Save and close the file.  Notice that we used some variable values there?  We need to add these into our variables.tf file.
+
+Go ahead and open it with your favorite text editor:
+
+```
+  $ vim variables.tf
+```
+
+And add in this content:
+
+variables.tf
+```
+variable "access_key" {}
+variable "secret_key" {}
+variable "key_name" {}
+variable "region" {}
+variable "ami" {}
+variable "instance_type" {}
+```
+
+And, finally, we need to add in some values for these variables.
+
+Open up your testing.tfvars file
+
+```
+  $ vim testing.tfvars
+```
+
+And add in this content (substitute in the appropriate values for your AWS account, region, etc.)
+
+testing.tfvars
+```
+access_key = "my_aws_access_key"
+secret_key = "my_aws_secret_key"
+key_name = "my_aws_key_pair_name"
+region = "us-east-1"
+ami = "ami-fce3c696"
+instance_type = "t2.medium"
+```
+
+Save and close the file.
+
+Finally, let's define that output that will be expected by our .kitchen.yml.  Go ahead and open up your output.tf file.
+
+```
+$ vim output.tf
+```
+
+And add this content
+
+```
+output "public_dns" {
+  value = "${aws_instance.example.public_dns}"
+}
+```
+
+Ok!  Now we're ready to finally create some test kitchen instances!
+
+Go ahead and run
+```
+  $ bundle exec kitchen converge
+```
+
+And it should converge successfully and you should see output that includes
+
+```
+Outputs:
+
+  public_dns = your_aws_instance_public_ip
+```
+
+Alright, now let's create some specs!
+
+Go ahead and create a default spec file (we'll continue calling it default for now)
+
+```
+  $ touch test/integration/default/default_spec.rb
+```
+
+And before we add anything, let's first try running the specs as is
+
+```
+  $ bundle exec kitchen verify
+```
+
+And if it runs successfully, you should see output that includes
+
+```
+0 examples, 0 failures
+```
+
+Let's add some examples!
+
+Open up the file
+
+```
+  $ vim test/integration/default/default_spec.rb
+```
+
+And let's add in a very basic test to make sure we are running on an ubuntu system
+
+test/integration/default/default_spec.rb
+```
+describe command('lsb_release -a') do
+  its('stdout') { should match (/Ubuntu/) }
+end
+```
+
+Now save and close the file and run the specs
+
+```
+  $ bundle exec kitchen verify
+```
+
+And hey, it passed!  You should see output that includes
+
+```
+Command lsb_release -a
+  stdout
+    should match /Ubuntu/
+```
+
+It's nice that it passed...but we still need to make sure that it fails to be certain it tests what we think it tests.  Let's try changing our terraform config to spin up an Amazon Linux System, rather than an Ubuntu system, and make sure that this test fails.
```

Comment:
This content should be simplified by starting with an Amazon AMI and then switching the configuration to Ubuntu after witnessing the failing test.